test/unit/step.spec.js

```diff
@@ -155,28 +163,20 @@ describe('Tour | Step', () => {
         id: 'test',
         scrollTo: true,
         text: 'This is a step for testing',
-        popperOptions: {
-          modifiers: [{ name: 'foo', options: 'bar' }]
+        floatingUIOptions: {
+          middleware: [defaultOffsetMiddleware, fooMiddleware],
         },
         showOn,
         when
       });
     });
 
-    it('allows the step to override a previously defined modifier', () => {
+    // @todo decide what to do here.
+    it.skip('allows the step to override a previously defined modifier', () => {
```

Comment:
Yeah wanted to talk about this one, since the format of the middleware changed, and that middleware are made to be composable, does it still make sense to allow overrides and instead just add more middlewares to do what's necessary?

Reply:
Unskiped the test this is the only test failure left where I think we might need to discuss options.